cacheable/README.md

```diff
@@ -118,6 +120,13 @@ Create a `LRU` object, all items will be cached in it.
 
     -   `False`: return reference of cached item
 
+-   `mutex_update`: mutex updating the cache if two or more callers
```

Comment:
allows only one thread to update the cache?

Reply:
1